exercises/practice/difference-of-squares/difference_of_squares.odin

```diff
@@ -0,0 +1,14 @@
+package difference_of_squares
+
+square_of_sum :: proc(n: int) -> int {
```

Comment:
This should be the student stub. It looks like this is the example solution.

Reply:
😳 Oops, yes I think this got overwritten by the test shell and I missed it – that script needs a fix too

Reply:
The test script had a bug where it clobbered student stubs if the test failed. This has been fixed.

Reply:
This file still needs to revert to an actual stub :)

Reply:
Oh man, this has tripped me up too many times now 🙈. I'll fix this, and add a check to `bin/run-test.sh` that _fails_ the test if the stub solution passes.

Reply:
I added a test that fails if the stub turns out to be a valid solution